charts/adot-exporter-for-eks-on-ec2/templates/adot-collector/configmap.yaml

```diff
@@ -9,39 +9,60 @@ metadata:
+            scrape_timeout: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeTimeout }}
+          scrape_configs: {{ .Values.adotCollector.daemonSet.ampreceivers.scrapeConfigs | nindent 10 }}
     processors:
       batch/metrics:
         timeout: {{ .Values.adotCollector.daemonSet.processors.timeout }}
     exporters:
       awsemf:
-        namespace: {{ .Values.adotCollector.daemonSet.exporters.namespace }}
+        namespace: {{ .Values.adotCollector.daemonSet.cwexporters.namespace }}
         log_group_name: '/aws/containerinsights/{{ .Values.clusterName }}/performance'
-        log_stream_name: {{ .Values.adotCollector.daemonSet.exporters.logStreamName }}
+        log_stream_name: {{ .Values.adotCollector.daemonSet.cwexporters.logStreamName }}
         resource_to_telemetry_conversion:
-          enabled: {{ .Values.adotCollector.daemonSet.exporters.enabled }}
-        dimension_rollup_option: {{ .Values.adotCollector.daemonSet.exporters.dimensionRollupOption }}
-        parse_json_encoded_attr_values: {{- range .Values.adotCollector.daemonSet.exporters.parseJsonEncodedAttrValues }}
+          enabled: {{ .Values.adotCollector.daemonSet.cwexporters.enabled }}
+        dimension_rollup_option: {{ .Values.adotCollector.daemonSet.cwexporters.dimensionRollupOption }}
+        parse_json_encoded_attr_values: {{- range .Values.adotCollector.daemonSet.cwexporters.parseJsonEncodedAttrValues }}
         - {{.}}{{- end }}
         metric_declarations:
           {{ .Values.adotCollector.daemonSet.metricDeclarations | nindent 10 }}
+      awsprometheusremotewrite:
+        namespace: {{ .Values.adotCollector.daemonSet.ampexporters.namespace }}
+        endpoint: {{ .Values.adotCollector.daemonSet.ampexporters.endpoint }}
+        # resource_to_telemetry_conversion: 
+        #   enabled: {{ .Values.adotCollector.daemonSet.ampexporters.resourcetotel }}
+        aws_auth:
+          region: {{ .Values.adotCollector.daemonSet.ampexporters.region }}
+          service: {{ .Values.adotCollector.daemonSet.ampexporters.service }}
+        
     service:
       pipelines:
         metrics:
           receivers: {{- range .Values.adotCollector.daemonSet.service.metrics.receivers }}
-          - {{.}}{{- end }}
+          - {{.}} {{- end }}
           processors: {{- range .Values.adotCollector.daemonSet.service.metrics.processors }}
-          - {{.}}{{- end }}
+          - {{.}} {{- end }}
+          {{- if eq .Values.adotCollector.daemonSet.cwEnabled false }}
+          exporters: {{ .Values.adotCollector.daemonSet.service.metrics.ampexporter }}
+          {{- else if eq .Values.adotCollector.daemonSet.ampEnabled false }}
+          exporters: {{ .Values.adotCollector.daemonSet.service.metrics.cwexporter }}
+          {{- else }}
           exporters: {{- range .Values.adotCollector.daemonSet.service.metrics.exporters }}
-          - {{.}}{{- end }}
+          - {{.}} {{- end }}
```

Comment:
Can we just range over the list of exporters here?  This logic is convoluted and probably unnecessary.

Reply:
Yeah, that is probably a much cleaner way to do it. I'll update the ReadMe to specify that the user can control the pipelines by changing the configuration of the helm chart.